mitiq/folding_cirq.py

```diff
@@ -0,0 +1,61 @@
+import random
+import cirq
+
+def local_folding(circuit, stretch, sampling=False):
```

Comment:
Missing type annotations (I'll suppress this comment for the rest though it applies in other places)